fixtures/dom/src/components/fixtures/text-inputs/index.js

```diff
@@ -110,6 +114,37 @@ class TextInputFixtures extends React.Component {
           <InputTestCase type="url" defaultValue="" />
         </TestCase>
 
+        <TestCase
```

Comment:
For new test cases that are stateful, can you break it out into it's own component? See [NumberInputExtraZeroes.js](https://github.com/facebook/react/blob/master/fixtures/dom/src/components/fixtures/number-inputs/NumberInputExtraZeroes.js) as an example.

Reply:
I was in doubt about the implementation. Thanks for the guideline!